deployment/distribution/src/main/resources/README.md

```diff
@@ -0,0 +1,95 @@
+  * Only MUSTs from the SAML V2.0 Standard Specification are tested.
+  * This test kit includes built-in support for Keycloak and Distributed Data Framework (DDF).
+
+Getting Started
+===============
+For a SAML CTK source distribution, please read the README.md file at
+https://github.com/connexta/saml-conformance for instructions on building this test kit.
+
+How to Run
+==========
+* Unzip the distribution.
+* Run the executable at <distribution_home>/bin/samlconf (*NIX) or <distribution_home>/bin/samlconf.bat (Windows)
+
+The `samlconf` script may take the following parameters:
+
+    NAME
+           samlconf - Runs the SAML Conformance Tests against an IdP
+
+    SYNOPSIS
+           samlconf [arguments ...]
+
+    DESCRIPTION
+           Runs the SAML Conformance Tests which test the compliance of an IdP
+           to the SAML Specifications. If a compliance issue is identified, a
+           SAMLComplianceException will be thrown with an explanation of the error and a direct
+           quote from the specification. Tests will not run if the corresponding
+           endpoints do not exist in the IdP's metadata. All of the parameters
+           are optional and if they are not provided, the default values will use
+           Distributed Data Framework (DDF)'s parameters. See the README.md file at
+           https://github.com/codice/ddf to learn more about DDF.
+
+    OPTIONS
+           -debug
+                Enables debug mode which enables more logging. This mode is off by default.
+
+           -ddf
+                Runs the DDF profile. If provided runs the optional SAML V2.0 Standard
+                Specification rules required by DDF.
+
+           -i path
+                The path to the directory containing the implementation's plugin and metadata.
+                The default value is /implementations/ddf.
+
+           -l
+                When an error occurs, the SAML V2.0 Standard Specification requires an IdP to
+                respond with a 200 HTTP status code and a valid SAML response containing an
+                error <StatusCode>.
+                If the -l flag is given, this test kit will allow HTTP error status codes as
+                a valid error response (i.e. 400's and 500's).
+                If it is not given, this test kit will only verify that a valid SAML error
+                response is returned.
+
+           -u username:password
+                The username and password to use when logging in.
+                The default value is admin:admin.
+
+
+Identity Provider Support
+=========================
+This test kit includes built-in support for Keycloak and Distributed Data Framework (DDF).
+To run the test against Keycloak:
+  * Setup your Keycloak IdP.
+  * Configure your IdP with the test kit's Service Provider (SP) metadata from `<distribution_home>/conf/samlconf-sp-metadata.xml`.
+  * Run the `samlconf` script with `-i <distribution_home>/implementations/keycloak`.
```

Comment:
✏️ Maybe include a sentence saying setup for DDF is the same except that no `-i` is needed?